docs/user/get-started.md

```diff
@@ -167,169 +137,21 @@ This chapter shows how to run Vald with fashion-mnist dataset.
-    - Loading from fashion-mnist dataset and set id for each vector that is loaded. This step will return the training dataset, test dataset, and ids list of ids when loading is completed with success.
-        <details><summary>example code</summary><br>
-
-        ```go
-        ids, train, test, err := load(datasetPath)
-        if err != nil {
-            glg.Fatal(err)
-        }
-        ```
-        </details>
-    3. Create the gRPC connection and Vald client with gRPC connection.
-        <details><summary>example code</summary><br>
-
-        ```go
-        ctx := context.Background()
-
-        conn, err := grpc.DialContext(ctx, grpcServerAddr, grpc.WithInsecure())
-        if err != nil {
-            glg.Fatal(err)
-        }
-
-        client := vald.NewValdClient(conn)
-        ```
-        </details>
-    4. Insert and Index
-    - Insert and Indexing 400 training datasets to the Vald agent.
-        <details><summary>example code</summary><br>
-
-        ```go
-        for i := range ids [:insertCount] {
-            if i%10 == 0 {
-                glg.Infof("Inserted %d", i)
-            }
-            _, err := client.Insert(ctx, &payload.Object_Vector{
-                Id: ids[i],
-                Vector: train[i],
-            })
-            if err != nil {
-                glg.Fatal(err)
-            }
-        }
-        ```
-        </details>
-    - Wait until indexing finish.
-        <details><summary>example code</summary><br>
-
-        ```go
-        glg.Info("Wait for indexing to finish")
-        time.Sleep(time.Duration(indexingWaitSeconds) * time.Second)
-        ```
-        </details>
-    5. Search
-    - Search 10 neighbor vectors for each 20 test datasets and return list of neighbor vector.
-    - When getting approximate vectors, the Vald client sends search config and vector to the server via gRPC.
-        <details><summary>example code</summary><br>
-
-        ```go
-        glg.Infof("Start search %d times", testCount)
-        for i, vec := range test[:testCount] {
-            res, err := client.Seach(ctx, &payload.Search_Request){
-                Vector: vec,
-                Config: &payload.Search_Config{
-                    Num: 10,
-                    Radius: -1,
-                    Epsilon: 0.01,
-                }
-            }
-            if err != nil {
-                glg.Fatal(err)
-            }
-
-            b, _ := json.MarshalIndent(res.GetResults(), "", " ")
-            glg.Infof("%d - Results : %s\n\n", i+1, string(b))
-            time.Sleep(1 * time.Second)
-        }
-        ```
-        </details>
+3. Perform a search action in Vald
+
+    In this example, the fashion-mnist dataset will insert into the vald cluster and perform a search using [vald-client-go](https://github.com/vdaas/vald-client-go).
+    Full example code is [`here`](../../example/client/main.go) and the explaination is [here](https://github.com/vdaas/vald-client-go/docs/users/code-explanation.md).
```

Comment:
- I'm not sure that this change is consistent with what we discussed 😕. where is `code-explanation.md`?

Reply:
It moved to go client library repo. I created a PR on that repo. 
Also I suggest to move the example program to that repo.